packages/charm/src/index.ts

```diff
@@ -38,7 +38,16 @@ export { type ParsedMention, type ProcessedPrompt } from "./imagine.ts";
 export { formatPromptWithMentions, parseComposerDocument } from "./format.ts";
 
 export const DEFAULT_MODEL = [
+  // "groq:llama-3.3-70b-specdec",
+  // "cerebras:llama-3.3-70b",
+  // "anthropic:claude-3-5-sonnet-latest",
   "anthropic:claude-sonnet-4-0",
+  // "gemini-2.0-flash",
+  // "gemini-2.0-flash-thinking",
+  // "gemini-2.0-pro",
+  // "o3-mini-low",
+  // "o3-mini-medium",
+  // "o3-mini-high",
```

Comment:
this doesn't seem like it should be coming back, we just removed it in https://github.com/commontoolsinc/labs/pull/1745/files

Reply:
thanks.. i made the same change in the background and rebase decided to go this way